slashCommands/vote.ts

```diff
@@ -12,178 +19,142 @@ export default {
                 type: ApplicationCommandOptionType.String,
                 name: "name",
                 description: "Give this vote a reason",
-                required: false
-            },
-            {
-                type: ApplicationCommandOptionType.Boolean,
-                name: "istimed",
-                description: "Makes this vote to a Timed Vote.",
-                required: false,
+                required: true
             },
             {
                 type: ApplicationCommandOptionType.Integer,
                 name: "votetime",
-                description: "Time the vote is running in seconds",
+                description: "Time the vote is running in seconds.",
                 required: false,
+                minValue: 0
             }
         ]
     },
     handler: async function(interaction: CommandInteraction) {
 
         // --- Variables
-        let maxLoop = 20;
 
         const upEmo = "👍";
         const downEmo = "👎";
 
         let usedVotes = 0;
-
         let msg = "";
 
-        let reactionAr = new Array(0);
 
         // --- Variables with input
         const options = interaction.options as CommandInteractionOptionResolver;
+        const time = options.getInteger("votetime", false);
+        const timed = time != undefined;
+        const title = options.getString("name", true);
 
-        let time = options.getInteger("votetime", false);
-
-        let timed = options.getBoolean("istimed");
-        timed = (timed == null) ? false : timed;
-
-        let title = options.getString("name");
-
-        // --- Check for undefined
-
-        time = ((time == undefined || time <= 0) ? 30 : time);
-
-        title = (title == undefined) ? "" : title;
-
-        // --- group variables
 
         const guid = get("vote_group", "config") as string;
```

Comment:
"vote_group" isn't in the config-templates - and should also use camelCase

Reply:
you mean voteGroup? The other config variables also follow x_x like announcement_channel

Reply:
I added it for now to the template the way it is

Reply:
Ah, yeah, thought all our config variables are camel-cased, turns out, they arent